.github/workflows/deploy-earn-protocol-landing-page-new-production.yaml

```
@@ -0,0 +1,153 @@
+          path: ${{ github.workspace }}/apps/earn-protocol-landing-page/.next/cache
+          key:
+            ${{ runner.os }}-earn-protocol-landing-page-app-${{ hashFiles('pnpm-lock.yaml') }}-${{
+            hashFiles('apps/earn-protocol-landing-page/**/*.ts',
+            'apps/earn-protocol-landing-page/**/*.tsx') }}
+          restore-keys:
+            ${{ runner.os }}-earn-protocol-landing-page-app-${{ hashFiles('pnpm-lock.yaml') }}-
+
+      - name: Extract commit hash
+        id: vars
+        shell: bash
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)